run-task.bat

```diff
@@ -1 +1 @@
-docker run --rm -v %~dp0:/mnt -p 8080:8080 --name odkx-docs odkx-docs %1
+docker run --rm -v "%~dp0:/mnt" -p 8080:8080 --name odkx-docs odkx-docs %1
```

Comment:
Can you submit your docker change in a separate PR? It's best to separate issues, the top part solve issue #174 the bottom part is different. You can create different pull requests by having different branches.

Reply:
Seen, will sort it out. Thanks.